templates/named.conf.erb

```diff
@@ -14,5 +14,15 @@ options  {
 include "<%= scope.lookupvar('::dns::localzonepath') %>";
 <% end -%>
 
+<%- if scope.lookupvar('::dns::acl_hash') and !scope.lookupvar('::dns::acl_hash').empty? -%>
```

Comment:
Keeping the check for .empty? ensures we don't have the acl block in the config when the hash is empty.